Alpaca.Markets/Enums/TimeInForce.cs

```diff
@@ -45,6 +46,7 @@ public enum TimeInForce
 
     /// <summary>
     /// The order will become a limit order if a limit price is specified or a market order otherwise at market close.
+    /// Not supported for crypto trading.
```

Comment:
Please, use the `remarks` tag instead of placing this information into the `summary` tag - for better visibility in VS and auto-generated docs.